lib/tasks/pubmed.rake

```diff
@@ -10,4 +10,36 @@ namespace :pubmed do
     doc = client.fetch_records_for_pmid_list(pmids)
     puts doc # XML document
   end
+
+  desc 'Harvest using a text file with a list of publications by PubmedID for the supplied cap_profile_id'
+  task :pmid_profile_id_import, [:path_to_report,:cap_profile_id] => :environment do |_t, args|
+    lines = IO.readlines args[:path_to_report]
+    author = Author.find_by_cap_profile_id(args[:cap_profile_id])
```

Comment:
Just curious about whether it requires `args[:cap_profile_id].to_i` (maybe A.R. takes care of any String/Integer conversions).